x-pack/plugins/apm/public/components/shared/KueryBar/view.js

```diff
@@ -112,16 +113,25 @@ class KueryBarView extends Component {
             style={{ display: 'inline-block', marginTop: '10px' }}
             title={
               <div>
-                There&#39;s no APM index pattern with the title &#34;
-                {apmIndexPatternTitle}
-                &#34; available. To use the Query bar, please choose to import
-                the APM index pattern via the{' '}
-                <KibanaLink
-                  pathname={'/app/kibana'}
-                  hash={`/home/tutorial/apm`}
-                >
-                  Setup Instructions.
-                </KibanaLink>
+                <FormattedMessage
+                  id="xpack.apm.kueryBar.indexPatternMissingWarningMessage"
+                  defaultMessage="There&#39;s no APM index pattern with the title &#34;{apmIndexPatternTitle}&#34; available.
```

Comment:
please use **'** here instead **&#39;**
```suggestion
                   defaultMessage="There's no APM index pattern with the title &#34;{apmIndexPatternTitle}&#34; available.
```